src/main/java/tr/com/srdc/cda2fhir/util/EMFUtil.java

```diff
@@ -31,7 +35,8 @@ static private void putReferences(FeatureMap featureMap, Map<String, String> res
 			EStructuralFeature feature = entry.getEStructuralFeature();
 			if (feature instanceof EReference) {
 				AnyType anyType = (AnyType) entry.getValue();
-				if ("content".equalsIgnoreCase(feature.getName())) {
+
+				if (Arrays.stream(supportedTypes).anyMatch(feature.getName().toLowerCase()::equals)) {
```

Comment:
What type of magic is this?

Reply:
stackoverflow "how do I work with arrays in java" magic.  basically it returns more than just "content" HTML tags now, it's those 3 in the array.

Reply:
stack overflow ftw